src/src/com/tns/Platform.java

```diff
@@ -156,6 +158,9 @@ public static int init(Application application, ThreadScheduler threadScheduler,
 			Date lastModDate = new Date(f.lastModified());
 			logger.write("init time=" + (d.getTime() - lastModDate.getTime()));
 		}
+		
+		directBufferFactory = new DirectBufferFactory(logger);
+		directBufferFactory.startListener();
```

Comment:
Can we make this initialize upon demand? I mean why starting a new thread if we are not going to use it?